cmake/modules/SwiftComponents.cmake

```diff
@@ -162,7 +162,27 @@ function(swift_install_in_component)
     return()
   endif()
 
-  install(${ARGN})
+  if((NOT SWIFT_INSTALL_AS_SYMLINKS) OR
+     (NOT ARG_DESTINATION) OR
+     (ARG_RENAME))
```

Comment:
RENAME seems to only be used by test suite, but where does `NOT ARG_DESTINATION` occur in practice?

Reply:
I don't think it ever occurs, but if someone didn't specify a destination and relied on CMake's default locations we'd need to add more logic here. I could make it fail loudly instead of falling back to a regular install if you think that's better.

Reply:
Maybe a warning?